src/main/java/com/google/api/codegen/transformer/java/JavaTestTransformer.java

```diff
@@ -86,11 +94,41 @@ private ApiTestClassView createTestClassView(Interface service) {
     testClass.apiClassName = surfaceNamer.getApiWrapperClassName(service);
     testClass.name = testClass.apiClassName + "Test";
     testClass.mockServiceClassName = "Mock" + testClass.apiClassName;
-    testClass.testCases = new ArrayList<>();
+    testClass.testCases = createTestCaseViews(service);
 
     String outputPath = pathMapper.getOutputPath(service, apiConfig);
     testClass.outputPath = outputPath + "/" + testClass.name + ".java";
     testClass.imports = typeTable.getImports();
     return testClass;
   }
+
+  private List<ApiTestCaseView> createTestCaseViews(Interface service) {
```

Comment:
I would recommend using a context parameter. That would make this code more "movable" in the future, since it wouldn't be tied to the member variables of this class.